ci/benchmark_parser.py

```diff
@@ -177,7 +220,14 @@ def check_mandatory_args(input_args):
     raw_results = pathlib.Path(args.results)
     if not args.key_sizes:
         print("Parsing benchmark results... ")
-        results = recursive_parse(raw_results, args.walk_subdirs, args.name_suffix)
+        HARDWARE_COST = None
```

Comment:
why use HARDWARE_COST in caps ? it has kind of a special meaning in python

Reply:
Apparently, the way I use it seems to looks like a Constant for pylint. Should I ignore the warning sent by the linter ?
I've to admit, this warning makes no sense to me though

Reply:
avoid using all caps if it's not supposed to be a runtime constant